src/Services/MailFinderService.php

```diff
@@ -79,13 +79,44 @@ public function findTemplateByTechnicalName(string $type, string $technicalName,
                 $supportedExtensions = $availableLoader->supportedExtensions();
 
                 foreach ($supportedExtensions as $supportedExtension) {
-                    foreach ($searchFolder as $folder) {
-                        $filePath = $path . '/email/' . $folder . '/' . $technicalName . '/' . $type . $supportedExtension;
-                        if (file_exists($filePath) && $content = $availableLoader->load($filePath)) {
-                            $this->fixTranslator($businessEvent);
-
-                            return $content;
-                        }
```

Comment:
Would be the same result not possible by pushing also this to the $searchFolder array? 🤔

Reply:
I don't think so.. because you'd never come to line 85 when having this structure:
 
custom/plugins/MyTheme/src/Resources/views/email/[SalesChannel ID]/[Language Code]/order_transaction.state.paid/html.twig
 
Or am I wrong? 🤔